odoo/modules/registry.py

```diff
@@ -682,6 +682,13 @@ def cursor(self):
             return TestCursor(self.test_cr, self.test_lock)
         return self._db.cursor()
 
+    def neutralize(self):
+        """ Neutralize database """
+        with self.cursor() as cr:
+            env = odoo.api.Environment(cr, SUPERUSER_ID, {})
+            for model in self:
+                env[model]._neutralize()
```

Comment:
You can also iterate on `env` to get models as recordsets:
```suggestion
            for model in env.values():
                model._neutralize()
```
BTW, `model_name` would be better than `model` in your version.